### PR TITLE
Add Graduation Information to Profile

### PR DESCRIPTION
**Pull request by sastorer**
Adds a Graduation Information header to the Profile page.

Graduation Information header includes Graduation Date.

### CODE REVIEW

File: resources/js/components/UserEditForm.vue [hunk 1]
```diff
@@ -357,13 +376,23 @@ export default {
         });
     },
   },
+  computed: {
+    graduationInfoRequired: function() {
+      if (this.user.primary_affiliation == "Student") {
+        return false;
+      } else {
+        return true;
+      }
```
Comment (JoshOldenburg) [inline resources/js/components/UserEditForm.vue:385]:
```suggestion
      return this.user.primary_affiliation === "student";
```

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]

File: resources/js/components/UserEditForm.vue [hunk 2]
```diff
@@ -357,13 +376,23 @@ export default {
         });
     },
   },
+  computed: {
+    graduationInfoRequired: function() {
+      if (this.user.primary_affiliation == "Student") {
+        return true;
+      } else {
+        return false;
+      }
```
Comment (JoshOldenburg) [inline resources/js/components/UserEditForm.vue:385]:
```suggestion
      return this.user.primary_affiliation === "student";
```

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]
Updating the suggestion

File: resources/js/components/UserEditForm.vue (hunk 1 above)
Comment (sastorer) [reply resources/js/components/UserEditForm.vue]: wouldn't it be
`return !this.user.primary_affiliation === "student";`

File: resources/js/components/UserEditForm.vue (hunk 2 above)
Comment (sastorer) [reply resources/js/components/UserEditForm.vue:385]: wouldn't it be
`return !this.user.primary_affiliation === "student";`

Comment (sastorer) [reply resources/js/components/UserEditForm.vue:385]: oh wait nvm

**Review by JoshOldenburg — APPROVED** [review]

**Review by kberzinch — APPROVED** [review]

**Pull request merged**